### PR TITLE
Context paths DBotScore and a few outputs hook validations

### PR DESCRIPTION
**Pull request by bakatzir**
@DeanArbel FYI

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16390

## Description
In order to make sure our context standard integrations changes for 5.0 will be top-notch,
added a few checks in our hook validations.
Please see error example in the referenced issue.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation - no need
- [ ] Code Review

### CODE REVIEW

File: Tests/scripts/hook_validations/integration.py [hunk 1]
```diff
@@ -87,12 +88,50 @@ def is_default_arguments(self):
...
+
+        return self._is_valid
+
     def is_valid_subtype(self):
         """Validate that the subtype is python2 or python3."""
         type_ = self.current_integration.get('script', {}).get('type')
         if type_ == 'python':
             subtype = self.current_integration.get('script', {}).get('subtype')
-            if not subtype or subtype not in ['python3', 'python2']:
+            if subtype not in ['python3', 'python2']:
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/integration.py]: let's move the subtype options to the constants.py file and use them as a set

Comment (bakatzir) [reply Tests/scripts/hook_validations/integration.py]: @yaakovi fixed also for the script.py

File: Tests/scripts/hook_validations/tests/integration_test.py [hunk 2]
```diff
@@ -701,3 +701,156 @@ def test_is_default_arguments_ok():
...
+                            "description": "The indicator type.",
+                            "type": "string"
+                        },
+                        {
+                            "contextPath": "DBotScore.Vendor",
+                            "description": "Vendor used to calculate the score.",
+                            "type": "string"
+                        },
+                        {
+                            "contextPath": "DBotScore.Sc0re",
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/tests/integration_test.py]:
```suggestion
                            "contextPath": "DBotScore.Score",
```

Comment (bakatzir) [reply Tests/scripts/hook_validations/tests/integration_test.py]: @yaakovi it should be sc0re
the assert is False

**Comment by yaakovi** [comment]
as we have a standard for the description syntax, we can verify them as well

**Comment by ronykoz** [comment]
We already talked about it, @bakatzir didn’t you say you will add it here?

**Comment by bakatzir** [comment]
@yaakovi @ronykoz 
indeed. Added that and treat a wrong description as warning, not as an error.

**Comment by bakatzir** [comment]
reference: https://github.com/demisto/etc/issues/17949

**Pull request merged**